frontend/src/app/main/exports.cljs

```diff
@@ -156,3 +156,35 @@
            :xmlnsXlink "http://www.w3.org/1999/xlink"
            :xmlns "http://www.w3.org/2000/svg"}
      [:& wrapper {:shape frame :view-box vbox}]]))
+
+;; TODO: unify with frame-svg?
+(mf/defc component-svg
```

Comment:
I thik it is ok to have a separated component for component :P

Reply:
Ok. They may diverge in the future.